NextGenConnect/JSON to HL7v2.xml

```diff
@@ -164,7 +166,10 @@ tmp[&apos;PID&apos;][&apos;PID.13&apos;][&apos;PID.13.6&apos;] = areaCode;
 tmp[&apos;PID&apos;][&apos;PID.13&apos;][&apos;PID.13.7&apos;] = number; 
 
 
-// PID-22 Ethnic Group -- MVS-528</script>
+// PID-22 Ethnic Group -- MVS-528
+
+tmp[&apos;PID&apos;][&apos;PID.22&apos;][&apos;PID.22.1&apos;] = patient[&apos;extension&apos;][1][&apos;extension&apos;][0][&apos;valueCoding&apos;][&apos;code&apos;];
+tmp[&apos;PID&apos;][&apos;PID.22&apos;][&apos;PID.22.3&apos;] = &apos;CDCREC&apos;;</script>
```

Comment:
Can you put the display value in the second component?

Reply:
No. It's required to be in the 3rd component.

Reply:
No, I mean the display value from extension.extension.valueCoding.display. It would be additional information, not changing what you have in the third component now.